includes/admin/tools/export/give-export-donations-exporter.php

```diff
@@ -663,6 +663,7 @@ public function print_csv_rows() {
 				foreach ( $row as $col_id => $column ) {
 					// Make sure the column is valid
 					if ( array_key_exists( $col_id, $cols ) ) {
+                        $column = maybe_serialize( $column );
```

Comment:
We need to document this change. So, can you please add a new unreleased tag (describing this change) just above the `@since  2.1` tag?

Would be something like this:

```
@unreleased Use maybe_serialize() for column data
```